app/views/newsletters/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="newsletter-text">
+  <p>
+    Want to be kept up to date with the latest news from Any Rat Rescue? 
+  </p>
+  <button class="subscribe"><%= link_to "Subscribe", "#" %></button>
+  <h1>
```

Comment:
@halogenandtoast I agree. The text I am referring to are subheadings and so don't really need to be h1. On looking at my other pages I realized that I was using h2 tags. I have changed it this way. I hope that makes sense.